common/src/main/java/com/microsoft/identity/common/internal/fido/PublicKeyCredentialRequestOptions.java

```diff
@@ -20,16 +20,32 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-package com.microsoft.identity.common.internal.fido
+package com.microsoft.identity.common.internal.fido;
+
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
+import lombok.Getter;
 
 /**
  * Representation of WebAuthn's PublicKeyCredentialRequestOptions.
  * https://w3c.github.io/webauthn/#dictdef-publickeycredentialrequestoptions
  */
-@kotlinx.serialization.Serializable
-data class PublicKeyCredentialRequestOptions(
-    val challenge: String,
-    val rpId: String,
-    val allowCredentials: List<PublicKeyCredentialDescriptor>,
-    val userVerification: String
-)
+@Getter
+public class PublicKeyCredentialRequestOptions {
+    public final String challenge;
+    public final String rpId;
+    public final List<PublicKeyCredentialDescriptor> allowCredentials;
+    public final String userVerification;
```

Comment:
Would be a good practice to put Serialized Name annotation on these

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2189/commits/2e2c99898db30576eca9f758a4fbf12395573103